spec/assaydepot_spec.rb

```diff
@@ -117,7 +116,7 @@
         end
 
         it "should have a description" do
-          provider_result["provider"]["description"].should_not be_nil
+          provider_result["provider"]["keywords"].should_not be_nil
```

Comment:
Totally cool, but we should probably change the spec to be `should have keywords`

Reply:
I used this:

```
        it "should have a description" do
          expect(provider_result['provider']).to have_key('keywords')
        end
```
If this is not the right right `matcher` please help!

Reply:
Your code is great, I was referring to the line above where it says `it "should have a description" do`, I'll see if I can setup a suggested change. It's purely cosmetic.

Reply:
your right! good catch.